src/components/email-flow/EmailFlowProof.tsx

```diff
@@ -1,15 +1,15 @@
-import { BodyText, HeaderText } from 'components/Text'
+import { BodyText, HeaderText } from 'components/ui/Text'
 import BadgeBlock from 'components/badges/BadgeBlock'
-import EmailProof from 'helpers/proofs/EmailProof'
 import MintedToken from 'models/MintedToken'
-import ReadyEmailProof from 'components/proofs/ReadyEmailProof'
+import Proof from 'components/proofs/Proof'
+import ProofModel from 'models/Proof'
 
 export default function ({
   proof,
   onMinted,
   onMintFailed,
 }: {
-  proof: EmailProof
+  proof: ProofModel
```

Comment:
Hmm, smells like you did use this args/props before in `BadgeBlock`, I think we need to extract this interface into a `models`

Reply:
I'm not sure if this is needed)